schemas/org.gnome.shell.extensions.pop-shell.gschema.xml

```diff
@@ -202,6 +202,16 @@
             <summary>Move window to the upper workspace</summary>
         </key>
 
+        <key type="as" name="pop-workspace-righ">
```

Comment:
These seem to take effect in a horizontal layout as well, which probably isn't desired behavior.